Description: Just set LFLAGS to -lasound, also rename it to LDFLAGS
Forwarded: yes
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@users.sourceforge.net>
--- a/modem/Makefile
+++ b/modem/Makefile
@@ -41,19 +41,14 @@
 
 ifdef SUPPORT_ALSA
 CFLAGS+= -DSUPPORT_ALSA=1
-ifeq ($(ARCH64), 0)
-LFLAGS+= /usr/lib/libasound.so
-else
-# LFLAGS+= /usr/lib32/libasound.so , replaced by -l commands search for asound
-LFLAGS+= -lasound
-endif
+LDFLAGS+= -lasound
 endif
 
 slmodemd:
-	$(CC) -o slmodemd modem_main.o $(all-objs) $(LFLAGS)
+	$(CC) -o slmodemd modem_main.o $(all-objs) $(LDFLAGS)
 
 modem_test:
-	$(CC) -o modem_test modem_test.o $(all-objs) $(LFLAGS)
+	$(CC) -o modem_test modem_test.o $(all-objs) $(LDFLAGS)
 
 clean:
 	$(RM) slmodemd modem_test modem_main.o modem_cmdline.o modem_test.o $(modem-objs) $(dp-objs) $(sysdep-objs)
